src/main/java/ru/clevertec/userservice/role/domain/Role.java

```diff
@@ -1,4 +1,41 @@
 package ru.clevertec.userservice.role.domain;
 
-public class Role {
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import ru.clevertec.userservice.user.domain.User;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.UUID;
+
+@Table(name = "user_role")
```

Comment:
if the class has name role then the table name must be role as well
user_role can be the table with two foreign key user_id and role_id